(2) Vox Populi/Database Changes/Civilizations/Carthage.sql

```diff
@@ -42,17 +42,15 @@ INSERT INTO Civilization_BuildingClassOverrides
 VALUES
 	('CIVILIZATION_CARTHAGE', 'BUILDINGCLASS_NATIONAL_TREASURY', 'BUILDING_GREAT_COTHON');
 
-UPDATE Building_ClassesNeededInCity
-SET BuildingClassType = 'BUILDINGCLASS_MARKET'
-WHERE BuildingType = 'BUILDING_GREAT_COTHON';
-
```

Comment:
Need to delete from `Building_ClassesNeededInCity` here, since EIC has a requirement.